src/api/app/controllers/webui/users_controller.rb

```diff
@@ -11,13 +11,24 @@ def index
   end
 
   def show
-    @iprojects = @displayed_user.involved_projects.pluck(:name, :title)
-    @ipackages = @displayed_user.involved_packages.joins(:project).pluck(:name, 'projects.name as pname')
-    @owned = @displayed_user.owned_packages
     @groups = @displayed_user.groups
     @role_titles = @displayed_user.roles.global.pluck(:title)
     @account_edit_link = CONFIG['proxy_auth_account_page']
 
+    if Flipper.enabled?(:user_profile_redesign, User.possibly_nobody)
+      attribute_type = AttribType.find_by_name!('OBS:OwnerRootProject')
+      @owner_root_project_exists = Project.find_by_attribute_type(attribute_type).exists?
+
+      filters = adjust_filters
+
+      @involved_items = @displayed_user.involved_items(filters)
+      @involved_items_as_owner = @displayed_user.involved_items_as_owner(filters) if @owner_root_project_exists
+    else
+      @iprojects = @displayed_user.involved_projects.pluck(:name, :title)
```

Comment:
Is this a new product from Apple? If not, I would go with something more explicit, although longer, like `@involved_projects`. Same for `@ipackages` which would be `@involved_packages`.

Reply:
This is for the old views, no need to change this. It'll be dropped once we roll this out.

Reply:
Oh, I totally overlooked that. :facepalm: